tortoise.py

```diff
@@ -1,27 +1,28 @@
 """Mini-language to control a simple turtle-graphics system."""
 
-import sys
 import tortoise_interpreter
+import argparse
+
 
 
 def main():
     """Read in a tortoise program to be interpreted."""
-    try:
-        filename = sys.argv[1]
-    except IndexError:
-        print(
-            "\nYou have to specify what tortoise you want executed.\
-              \nPlease add it as an argument on the command line.\n"
-        )
-        return
-
-    with open(filename, encoding="utf-8") as tortoise:
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("filename", help="To specify the file you want to use on the program")
```

Comment:
Just a wording thing, it would read better as "Specify the Tortoise file you want to run."